frontend/src/app/features/dashboard/timeline/timeline.component.ts

```diff
@@ -18,6 +17,10 @@ import { TimelineSection } from '#types/dashboard/timeline/timeline-section';
 })
 export class TimelineComponent extends BaseComponent implements OnInit, AfterViewInit, OnDestroy {
     public readonly defaultPageNumber = 12;
+    public readonly completedTask = TaskStatus.Completed;
```

Comment:
No need to put each value into field. You may create:
public readonly TaskStatus = TaskStatus;
and use it directly in html

Reply:
Fixed, thanks